README.md

```diff
@@ -26,7 +26,7 @@ line to your `Cargo.toml` (We recommend pinning to a specific version):
 
 ```toml
 [dependencies]
-openzeppelin-stylus = "0.1.1"
+openzeppelin-stylus = "0.2.0-alpha.1"
```

Comment:
I recommend we still keep the dependency to `0.1.1` in the readme on main, since it is what people usually follow and we don't want them to accidentally use an unaudited release.
We should(next week, after release) add a section that explains alpha releases and release cadence

Reply:
Fair enough, will change this.